pavelib/assets.py

```diff
@@ -775,7 +808,15 @@ def update_assets(args):
     execute_compile_sass(args)
 
     if args.collect:
-        collect_assets(args.system, args.settings)
+        if args.collect_log_dir:
+            collect_log_args.update({COLLECTSTATIC_LOG_DIR_ARG: args.collect_log_dir})
+
+        # Run collectstatic in debug mode if update_assets is in debug mode and no log location is specified
+        # This means collectstatic results will output to console
+        if args.debug and not args.collect_log_dir:
+            collect_log_args.update({COLLECTSTATIC_LOG_DIR_ARG: None})
```

Comment:
Clearer?

    if args.collect:
        kwargs = {}

        if args.debug:
            kwargs['log_dir'] = None

        if args.collect_log is not None:
            kwargs['log_dir'] = args.collect_log

        collect_assets(args.system. args.settings, **kwargs)